src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -674,6 +675,83 @@ class ObservationStore(
     }
   }
 
+  fun removePlotFromTotals(monitoringPlotId: MonitoringPlotId) {
+    val (plantingSiteId, plantingZoneId) =
+        dslContext
+            .select(
+                PLANTING_ZONES.PLANTING_SITE_ID.asNonNullable(), PLANTING_ZONES.ID.asNonNullable())
+            .from(MONITORING_PLOTS)
+            .join(PLANTING_SUBZONES)
+            .on(MONITORING_PLOTS.PLANTING_SUBZONE_ID.eq(PLANTING_SUBZONES.ID))
+            .join(PLANTING_ZONES)
+            .on(PLANTING_SUBZONES.PLANTING_ZONE_ID.eq(PLANTING_ZONES.ID))
+            .where(MONITORING_PLOTS.ID.eq(monitoringPlotId))
+            .fetchOne() ?: throw PlotNotFoundException(monitoringPlotId)
+
+    val negativeCountField = DSL.count().neg()
+    val negativeCounts:
```

Comment:
Took a little bit to digest! 

I think may read a little easier with a data class with the following:
```
observationId,
isPermanent
plantCountsBySpecies
```

Reply:
@tommylau523 Is it clearer now?

Reply:
Looks great! Thanks Steve it was a very nit comment in the first place.